pkg/health/health.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package health
+
+import (
+	ctx "context"
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+)
+
+func Health(context *k8sutil.Context, operatorNamespace, clusterNamespace string, exitOnError bool) {
```

Comment:
No need for the parameter, just define the local var to keep the method signature simpler. Or maybe this local var isn't even needed if each of these child methods can just pass `false`
```suggestion
func Health(context *k8sutil.Context, operatorNamespace, clusterNamespace string) {
   exitOnError := true
```

Reply:
got it